WordPressKit.xcodeproj/project.pbxproj

```diff
@@ -1550,6 +1552,30 @@
 			path = Plans;
 			sourceTree = "<group>";
 		};
+		3FFCC0532BABC75F0051D229 /* Sources */ = {
+			isa = PBXGroup;
+			children = (
+				3FFCC0542BABC7680051D229 /* APIInterface */,
+			);
+			path = Sources;
+			sourceTree = "<group>";
+		};
+		3FFCC0542BABC7680051D229 /* APIInterface */ = {
+			isa = PBXGroup;
+			children = (
+				3FFCC0562BABC7D20051D229 /* include */,
+			);
+			path = APIInterface;
+			sourceTree = "<group>";
+		};
+		3FFCC0562BABC7D20051D229 /* include */ = {
+			isa = PBXGroup;
+			children = (
+				3FFCC0552BABC78B0051D229 /* WordPressComRESTAPIInterfacing.h */,
+			);
+			path = include;
+			sourceTree = "<group>";
+		};
```

Comment:
I took the chance of adding this new file to setup the `Sources/` SPM structure. 

After this lands, I'll make a dedicated PR that moves all the files in `Sources/WordPressKit` and a new `Tests/WordPressKitTests`.

Once that happens, I'll follow up with another PR that extracts `WordPressComRestApi` and the related types in something I'm tentatively naming "CoreAPI", as per https://github.com/wordpress-mobile/WordPressKit-iOS/pull/738